bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -91,7 +90,7 @@ public void run() {
     }
   }
 
-  private static ExecutorService createExecutorService(BigtableOptions options) {
+  public static ExecutorService createExecutorService(BigtableOptions options) {
```

Comment:
You are right, "ExecutorService" alone is kind of generic. "WorkerExecutorService"?